sql/src/test/java/org/apache/druid/sql/calcite/SqlTestFrameworkConfig.java

```diff
@@ -120,7 +154,26 @@ private ConfigurationInstance getConfigurationInstance()
 
     ConfigurationInstance buildConfiguration(SqlTestFrameworkConfig config)
     {
-      return new ConfigurationInstance(config, testHost);
+      return new ConfigurationInstance(config, testHostSupplier.apply(createTempFolder("druid-test")));
+    }
+
+    protected File createTempFolder(String prefix)
+    {
+      File tempDir = FileUtils.createTempDir(prefix);
+      Runtime.getRuntime().addShutdownHook(new Thread()
```

Comment:
do we really need this? can this be replaced via a temporary folder or `deleteOnExit`?